activation/activation.go

```diff
@@ -506,11 +458,13 @@ func (b *Builder) PublishActivationTx(ctx context.Context) error {
 
 	challenge, err := b.loadChallenge()
 	if err != nil {
-		logger.With().Info("challenge not loaded", log.Err(err))
-		logger.With().Info("building new atx challenge", log.Stringer("current_epoch", b.currentEpoch()))
+		logger.With().Info("building new atx challenge",
+			log.Stringer("current_epoch", b.currentEpoch()),
+			log.Err(err),
```

Comment:
does this error need to be logged? Won't it just be `sql.NotFound` most of the time?

Reply:
i wanted to keep it, just in case